sdk/spring/azure-spring-cloud-autoconfigure/src/main/java/com/azure/spring/cloud/autoconfigure/cache/AzureRedisAutoConfiguration.java

```diff
@@ -34,12 +32,6 @@
 @ConditionalOnClass(RedisOperations.class)
 @EnableConfigurationProperties(AzureRedisProperties.class)
 public class AzureRedisAutoConfiguration {
-    private static final String REDIS = "Redis";
```

Comment:
There's no way to collect redis usage now.

Reply:
So, we will keep the telemetry usage of redis first?

Reply:
Let's remove the telemetry in this PR and figure out how to collect the Redis data later.